tests/testthat/_snaps/st_bonds.md

```diff
@@ -0,0 +1,11 @@
+# with no argument with multiple values errors gracefully
+
+    Must privide one argument of `run_stress_test()` with multiple values.
+    i Do you need to use `run_stress_test()` instead?
+
+# with more than one long stress-test argument errors gracefully
+
+    Must provide no more than one argument with multiple values.
+    x Arguments with multiple values: lgd_senior_claims, terminal_value.
+    i Did you forget to pick only one?
+
```

Comment:
These are the error messages the user will see. Please review and suggest the wording you prefer (ideally still following the tidyverse style guide for error messages).